apm-agent-plugins/apm-jaxrs-plugin/src/main/java/co/elastic/apm/agent/jaxrs/JaxRsTransactionNameInstrumentation.java

```diff
@@ -92,12 +94,16 @@ private static void setTransactionName(@SimpleMethodSignature String signature,
                 transaction.withName(transactionName, PRIO_HIGH_LEVEL_FRAMEWORK, false);
                 transaction.setFrameworkName(FRAMEWORK_NAME);
                 String version = versionsCache.get(javax.ws.rs.GET.class);
-                boolean isContains = versionsCache.containsKey(javax.ws.rs.GET.class);
-                if (version == null && !isContains) {
+                if (version == null) {
```

Comment:
Maybe implement the caching within `VersionUtils` so we don't have to duplicate it on every usage?

We could also implement a fallback mechanism to get the version:

```java
Package pkg = clazz.getPackage();
return (pkg != null ? pkg.getImplementationVersion() : null);
```